resources/views/settings/settings.blade.php

```diff
@@ -0,0 +1,79 @@
+@extends('layouts.main_layout')
+
+@section('head')
+
+    <title>Settings</title>
+
+@endsection
+
+@section('mainBody')
+
+    <main class="primary-main row">
+        <section class="grid register-main">
+
+            <form class="form-horizontal">
```

Comment:
So this form currently doesn't actually submit anything, but it seems like it's pretty close. You've got the basics set up, might as well wire them together. `action="/settings"` and `method="POST"` here, along with in the route `Route::post('/settings', SettingsController@create);` would pretty much do it.